pipeline/scripts/5/1/tier-1/test-upgrade-4x-to-5x-rgw-tests.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning RHCS 4x to 5x upgrade testing with RGW tests."
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.1"
+test_suite="suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml"
+test_conf="conf/pacific/rgw/upgrade_small-cluster.yaml"
+test_inventory="conf/inventory/rhel-8-latest.yaml"
+return_code=0
+
+# Process the CLI arguments for IBM-C environment
+CLI_ARGS=$@
+cloud="ibmc"
+if [ -z "${CLI_ARGS##*$cloud*}" ] ; then
+    test_inventory="conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+else
+    CLI_ARGS="$CLI_ARGS --post-results --report-portal"
+fi
+
+$WORKSPACE/.venv/bin/python run.py \
+    --osp-cred $HOME/osp-cred-ci-2.yaml \
+    --rhbuild $rhbuild \
+    --platform $platform \
+    --instances-name $instance_name \
+    --global-conf $test_conf \
+    --suite $test_suite \
+    --inventory $test_inventory \
+    --log-level DEBUG \
+    $CLI_ARGS
+
+if [ $? -ne 0 ]; then
+    return_code=1
+fi
+
+CLEANUP_ARGS="--log-level debug --osp-cred $HOME/osp-cred-ci-2.yaml"
+if [ -z "${CLI_ARGS##*$cloud*}" ] ; then
+    CLEANUP_ARGS="$CLEANUP_ARGS --cloud ibmc"
+fi
+
+$WORKSPACE/.venv/bin/python run.py --cleanup $instance_name $CLEANUP_ARGS
+
+if [ $? -ne 0 ]; then
+    echo "cleanup instance failed for instance $instance_name"
+fi
+
+exit ${return_code}
```

Comment:
The total runtime of the test is around 3 hours. Can we avoid running this test in the nightly pipeline (against all builds)? We can include this in the long running (cron pipeline) one... we expect it to be up shortly

Reply:
Removed sh file